pkgs/os-specific/linux/zfs/default.nix

```diff
@@ -180,10 +181,10 @@ in {
     # incompatibleKernelVersion = "4.19";
 
     # this package should point to a version / git revision compatible with the latest kernel release
-    version = "0.8.0-rc2";
+    version = "0.8.0-rc3";
 
-    rev = "af2e8411dacbc694b1aaf9074e68a9d12270e74c";
-    sha256 = "0wm7x9dwrw30jnjlnz6a224h88qd6a5794pzbjsih50lqb10g2gy";
+    rev = "9b626c126e78cdc36200b66c7cd1dc6a06cf400d";
```

Comment:
Remove this line.
Then, you should also update the `fetchpatch` part to use `https://github.com/Mic92/zfs/commit/10dba62167bc87852b8dab0d5e8159f10df5608f.patch`.
cc @Mic92

Reply:
What is the problem with the current patch url?
In any case we should use the right git tag. I created a new branch that is rebased onto `0.8.0-rc3` -> [nixos-zfs-2019-01-18](https://github.com/Mic92/zfs/tree/nixos-zfs-2019-01-18)

Reply:
The cuurent patch URL changes when changing `rev` but one may forget changing the hash (which changes due to the patch name change). There is no disadvantage in using a specific commit as patch as long as it applies cleanly.

Reply:
The has should change. Just put a nonsense hash there and you should see it change.

Reply:
I tried... The content didn't change, so the sha256 of the content also didn't change. It is saved to the store as a different derivation, but with the same hash.

Here, I can try it again:

```
trying https://github.com/Mic92/zfs/commit/bc29b5783da0af2c80c85126a1831ce1d52bfb69.patch                                                     |
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current                                                               |
                                 Dload  Upload   Total   Spent    Left  Speed                                                                 |
100  1280    0  1280    0     0   4076      0 --:--:-- --:--:-- --:--:--  4076                                                                |
hash mismatch in fixed-output derivation '/nix/store/hlgi8ymifwxrkba31qyjm03w6ih3fr0j-bc29b5783da0af2c80c85126a1831ce1d52bfb69.patch':        |
  wanted: sha256:1sdcr1w2jp3djpwlf1f91hrxxmc34q0jl388smdkxh5n5bpw5gzx                                                                         |
  got:    sha256:1sdcr1w2jp3djpwlf1f91hrxxmc34q0jl388smdkxh5n5bpw5gzw                                                                         |
cannot build derivation '/nix/store/02331g18cnflxd4sjq8mh5czm8cfwkp4-zfs-user-0.8.0-rc3.drv': 1 dependencies couldn't be built                
```

Reply:
Ah, I see.